codeOfConduct.md

```diff
@@ -0,0 +1,15 @@
+Harassment includes offensive verbal comments related to gender, gender identity and expression, age, sexual orientation, disability, physical appearance, body size, race, ethnicity, religion (or lack thereof), sexual images in public spaces, deliberate intimidation, stalking, following, harassing photography or recording, sustained disruption of talks or other events, inappropriate physical contact, and unwelcome sexual attention.
+
+Participants asked to stop any harassing behavior are expected to comply immediately.
+
+Sponsors are also subject to the anti-harassment policy. In particular, sponsors should not use sexualized images, activities, or other material. Booth staff (including volunteers) should not use sexualized clothing/uniforms/costumes, or otherwise create a sexualized environment.
+
+If a participant engages in harassing behavior, the conference organizers may take any action they deem appropriate, including warning the offender or expulsion from the conference with no refund.
+
+If you are being harassed, notice that someone else is being harassed, or have any other concerns, please contact a member of conference staff immediately. Conference staff can be identified as they'll be wearing branded t-shirts and badges.
+
+Conference staff will be happy to help participants contact venue security or local law enforcement, provide escorts, or otherwise assist those experiencing harassment to feel safe for the duration of the conference.
+
+We expect participants to follow these rules at conference and workshop venues and conference-related social events.
+
+This Code of Conduct is adapted from Conference Code of Conduct and React Rally Code of Conduct.
```

Comment:
I had set up a PR as well for the same new file (duplicate issues were created it looks like). In my changes I noticed that the links needed to be added in as well 👍.

```suggestion
This Code of Conduct is adapted from [Conference Code of Conduct](http://confcodeofconduct.com/) and [React Rally Code of Conduct](http://www.reactrally.com/#/conduct).
```